tests/unit_tests/assembly/global.rs

```diff
@@ -290,8 +290,10 @@ fn test_compute_global_potential() {
     }
 
     let global_potential = compute_global_potential(&MockScalarElementAssembler).unwrap();
-
     assert_scalar_eq!(global_potential, 9.0, comp = float);
+
+    let par_global_potential = par_compute_global_potential(&MockScalarElementAssembler).unwrap();
+    assert_scalar_eq!(par_global_potential, 9.0, comp = float);
 }
```

Comment:
Maybe move the mock assembler out of the test and put the parallel computation in its own test? Would be useful in order to be able to quickly spot tests that deal with parallelism (i.e. the test name would have "par" in it)

Reply:
Makes sense